web/packages/core/src/public/v1.ts

```diff
@@ -0,0 +1,173 @@
+import { MovieMetadata } from "../movie-metadata";
+import { DataLoadOptions, URLLoadOptions } from "../load-options";
+import { ReadyState } from "../internal/player/inner";
+
+export interface PlayerV1 {
+    /**
+     * A movie can communicate with the hosting page using fscommand
+     * as long as script access is allowed.
+     *
+     * @param command A string passed to the host application for any use.
+     * @param args A string passed to the host application for any use.
+     * @returns True if the command was handled.
+     */
+    onFSCommand: ((command: string, args: string) => boolean) | null;
+
+    /**
+     * Any configuration that should apply to this specific player.
+     * This will be defaulted with any global configuration.
+     */
+    config: URLLoadOptions | DataLoadOptions | object;
+
+    /**
+     * The effective config loaded with the last call to `load()`.
+     * If no such call has been made, this will be `null`.
+     */
+    readonly loadedConfig: URLLoadOptions | DataLoadOptions | null;
+
+    /**
+     * Indicates the readiness of the playing movie.
+     *
+     * @returns The `ReadyState` of the player.
+     */
+    get readyState(): ReadyState;
+
+    /**
+     * The metadata of the playing movie (such as movie width and height).
+     * These are inherent properties stored in the SWF file and are not affected by runtime changes.
+     * For example, `metadata.width` is the width of the SWF file, and not the width of the Ruffle player.
+     *
+     * @returns The metadata of the movie, or `null` if the movie metadata has not yet loaded.
+     */
+    get metadata(): MovieMetadata | null;
+
+    /**
+     * Reloads the player, as if you called {@link RufflePlayer.load} with the same config as the last time it was called.
+     *
+     * If this player has never been loaded, this method will return an error.
+     */
+    reload(): Promise<void>;
+
+    /**
+     * Loads a specified movie into this player.
+     *
+     * This will replace any existing movie that may be playing.
+     *
+     * @param options One of the following:
+     * - A URL, passed as a string, which will load a URL with default options.
+     * - A [[URLLoadOptions]] object, to load a URL with options.
+     * - A [[DataLoadOptions]] object, to load data with options.
+     * The options, if provided, must only contain values provided for this specific movie.
+     * They must not contain any default values, since those would overwrite other configuration
+     * settings with a lower priority (e.g. the general RufflePlayer config).
+     *
+     * The options will be defaulted by the [[config]] field, which itself
+     * is defaulted by a global `window.RufflePlayer.config`.
+     */
+    load(options: string | URLLoadOptions | DataLoadOptions): Promise<void>;
+
+    /**
+     * Plays or resumes the movie.
+     */
+    play(): void;
```

Comment:
This is actually Ruffle's suspend/resume, not Flash's play/pause. I suggest we rename this to `resume()` or `suspended = false` or something similar